linux-anvil-aarch64-cuda/Dockerfile

```diff
@@ -35,12 +35,7 @@ ADD https://loripsum.net/api /opt/docker/etc/gibberish
 # Naming convention changed with cos8 - see:
 # * https://lists.centos.org/pipermail/centos-devel/2019-September/017847.html
 # * https://www.centos.org/keys/#project-keys
-RUN if [ "$CENTOS_VER" -le "7" ]; then \
-        rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-${CENTOS_VER} && \
-        rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-${CENTOS_VER}-aarch64; \
-    else \
-        rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-centosofficial; \
-    fi
+RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
```

Comment:
Probably need the same thing here

https://github.com/conda-forge/docker-images/blob/759e147952e4da0413dacc007e843871398354aa/linux-anvil-ppc64le-cuda/Dockerfile#L55-L60

Maybe we can update the comment above as well

Reply:
@jakirkham Can you offer guidance on what you'd put in the comment? I'm just not familiar enough to know what should be referenced here, if anything.

Oops, I meant to commit the same for ppc64le. Will fix.

Also, I verified that there are no architecture-specific signing keys for aarch64 like `/etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-${CENTOS_VER}-aarch64` in the previous code.